agent/tool-scripts/jstack

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# The following script collect heap used stats for jvm and containerized jvm
```

Comment:
Heap?

Seems like the same comments that were made for the jmap PR #809 apply here.  Can you be sure to apply those first to that tool, get that merged, and then we can do the same with this one?